plugins/sip/lib/model/providers/pexip/kPexipUtils.php

```diff
@@ -423,10 +419,33 @@ public static function extractIdFromCreatedResult($result,$url ,$headerSize)
 				return $virtualRoomId;
 			}
 		}
-		KalturaLog::info('Could not extract ID from headers');
+
+		KalturaLog::info("Could not extract ID from headers with url: $url");
 		return null;
 	}
 
+	/**
+	 * @param $result
+	 * @param $url
+	 * @param $headerSize
+	 * @return int|null
+	 */
+	public static function extractIdFromCreatedResult($result, $url ,$headerSize)
+	{
+		$header = substr($result, 0, $headerSize);
+		$headerData = explode('\n', $header);
+		KalturaLog::info('Checking Headers ' . print_r($headerData, true));
+		$virtualRoomId = self::getVirtualRoomId($url, $headerData);
+		if (is_null($virtualRoomId))
```

Comment:
Why not add this attempt to obtain the ID into `getVirtualRoomId()` (in the event the first detection method failed)?